include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -4670,6 +4671,41 @@ struct OrtApi {
                   _In_reads_(num_external_initializer_files) char* const* external_initializer_file_buffer_array,
                   _In_reads_(num_external_initializer_files) const size_t* external_initializer_file_lengths,
                   size_t num_external_initializer_files);
+
+  /** \brief Create an OrtLoraAdapter
+   *
+   * The function attempts to locate file specified by adapter_file_path, read it and create an OrtLoraAdapter
+   * instance. The adapter_file_path should be a valid absolute path to a file that contains a valid Lora Adapter
+   * format. The function attempts to validate the format at load time. The file will always be memory mapped, unless
+   * the platform does not support memory mapping, in which case the file will be read into memory.
+   *
+   * \param[in] adapter_file_path Absolute file path to the adapter file.
```

Comment:
what's the reason for not also allowing relative paths here? is this constraint inconsistent other path parameters, like CreateSession()'s model_path?

Reply:
It was a security consideration. It also a part of the design.